databricks/koalas/series.py

```diff
@@ -2294,6 +2294,56 @@ def apply(self, func, args=(), **kwds):
         wrapped = ks.pandas_wraps(return_col=return_sig)(apply_each)
         return wrapped(self, *args, **kwds).rename(self.name)
 
+    # TODO: not all arguments are implemented comparing to Pandas' for now.
+    def aggregate(self, func: Union[str, List[str]]):
+        """Aggregate using one or more operations over the specified axis.
+
+        Parameters
+        ----------
+        func : str or a list of str
+            function name(s) as string apply to series.
+
+        Returns
+        -------
+        scalar, Series
+            The return can be:
+            - scalar : when Series.agg is called with single function
+            - Series : when Series.agg is called with several functions
+
+        Notes
+        -----
+        `agg` is an alias for `aggregate`. Use the alias.
+
+        See Also
+        --------
+        databricks.koalas.Series.apply
+        databricks.koalas.Series.transform
+
+        Examples
+        --------
+        >>> s = ks.Series([1, 2, 3, 4])
+        >>> s.agg('min')
+        1
+
+        >>> s.agg(['min', 'max'])
+        min    1
+        max    4
+        Name: 0, dtype: int64
+        """
+        if isinstance(func, list):
+            if all((isinstance(f, str) for f in func)):
+                rows = OrderedDict((f, eval("self.{}()".format(f), dict(self=self))) for f in func)
+                return Series(rows)
```

Comment:
This runs Spark jobs many times. In this case, I think we can reuse `DataFrame`'s aggregate.

Reply:
@ueshin Thanks again!! fixed it :)
Anyway I have a question, (It may be a very basic question though 😿 )
is it right the every function call at eval in OrderedDict comprehension call spark job each time?

Reply:
yes, each aggregate function for Series call triggers `sdf.head(2)` in https://github.com/databricks/koalas/blob/cbcb502e12df87cd948ed6ce3c23ece8cb342ba9/databricks/koalas/series.py#L3139-L3149 to return the result value.

Reply:
@ueshin Oh now i totally got it. Thanks ueshin!! 😃